lib/routes/api/repo.js

```diff
@@ -253,12 +261,17 @@ exports.create_project = function(req, res, next) {
   Project.findOne({ name: name }, projectResult);
 }
 
-/*
- * DELETE /:org/:repo
- *
- * @param url Url of repository to delete. Also archives all jobs (marks as archived in DB which makes them hidden).
+/**
+ * @api {delete} /:org/:repo Delete Repo
+ * @apiUse ProjectReference
+ * @apiPermission ProjectAdmin
+ * @apiDescription Deletes a repository/project.  Also archives all jobs (marks as archived in DB which makes them hidden).
```

Comment:
For your consideration: https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&es_th=1&ie=UTF-8#q=how%20many%20spaces%20after%20a%20period&es_th=1

Still, I can certainly change this if you'd like.